README.md

```diff
@@ -124,6 +125,16 @@ For details on prompt registry client, refer to this [document](https://github.c
 We have created a sample project demonstrating the different clients' usage of the SAP Cloud SDK for AI for TypeScript/JavaScript.
 The [project README](https://github.com/SAP/ai-sdk-js/blob/main/sample-code/README.md) outlines the set-up needed to build and run it locally.
 
+## Deprecated Models
+
+The following models are deprecated and should not be used.
+Update your code to use one of the recommended replacement models.
+
+**text-embedding-ada-002**
+
+- Retirement Date: 2025-10-03
+- Replacement Models: `text-embedding-3-small` or `text-embedding-3-large`
```

Comment:
[pp] Can we have a table for this? I would assume we will have many in the future.
```suggestion
| Model Name               | Replacement                                        |
|--------------------------|----------------------------------------------------|
| `text-embedding-ada-002` | `text-embedding-3-small`, `text-embedding-3-large` |
```

Leave the retirement date out to avoid double maintaining and put the SAP note here https://me.sap.com/notes/3437766.